JSDemos/Demos/TreeView/TreeViewWithSearchBar/description.md

```diff
@@ -1 +1,11 @@
-This demo demonstrates the TreeView component&rsquo;s searching capabilities. The component can find nodes containing your search string or&nbsp;only starting with it&nbsp;depending on&nbsp;the search mode.
\ No newline at end of file
+To add a search bar to the TreeView and customize the search functionality, do the following:
+
+- Set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchEnabled) property to **true**.
+
+- Use the [searchMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchMode) property to specify whether items should contain (default), start with, or match the search string. In this example, you can switch between search modes in the drop-down menu.
+
+- Configure the [searchExpr](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchExpr) property to specify custom search data fields. The default search field is [text](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/#text). 
```

Comment:
`custom search data fields` Why do you add 'data' here? 

```suggestion
- Configure the [searchExpr](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchExpr) property to specify custom search fields. The default search field is [text](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/#text). 
```

Reply:
To highlight that these fields belong to data objects.